src/main.go

```diff
@@ -47,7 +46,7 @@ func main() {
 	wg.Add(1)
 
 	go func() {
-		parse.ParseStaticFile("bgptest1", msgChannel)
+		//parse.ParseStaticFile("bgptest1", msgChannel)
```

Comment:
actually ... should this call `parse.ParseRisLiveData(msgChannel, configStruct)`? Otherwise I don't see where the connections are opened